src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -153,6 +145,7 @@ public double fpsToRPM(double fps){
   @Override
   public void periodic() {
     // This method will be called once per scheduler run
-
+   new DriveCommand();
```

Comment:
I'd rather this be added to the scheduler rather than a new DriveCommand object being created and then garbage collected every periodic cycle.